indico/modules/rb_new/client/js/common/bookings/BookingEdit.jsx

```diff
@@ -277,32 +386,120 @@ class BookingEdit extends React.Component {
                     <Popup trigger={<Icon name="info circle" className="legend-info-icon" />}
                            position="right center"
                            content={<TimelineLegend labels={legendLabels} compact />} />
-                    <Label.Group styleName="occurrence-count">
-                        <Popup trigger={<Label color="blue" size="tiny" content={this.numberOfOccurrences} circular />}
-                               position="bottom center"
-                               on="hover">
+                    {this.renderNumberOfOccurrences()}
+                </Header>
+                {this.isOngoingBooking && timeChanged && (
+                    <Message styleName="ongoing-booking-explanation" color="green" icon>
+                        <Icon name="code branch" />
+                        <Message.Content>
                             <Translate>
-                                Number of occurrences for this booking
+                                Your booking has already started and will be split into:
                             </Translate>
-                        </Popup>
-                        {numNewCandidates > 0 && (
-                            <>
-                                {' → '}
-                                <Popup trigger={<Label color="grey" size="tiny" content={numNewCandidates}
+                            <List bulleted>
+                                <List.Item>
+                                    <Translate>
+                                        the original booking, which will be shortened;
```

Comment:
i think the comma before `which` should go away here or be added in the second line as well. dunno which way is more correct ;)